test/integration/image-component/default/test/index.test.js

```diff
@@ -1039,6 +1040,26 @@ function runTests(mode) {
       }
     }
   })
+
+  it('should be valid W3C HTML', async () => {
+    let browser
+    try {
+      browser = await webdriver(appPort, '/valid-html-w3c')
```

Comment:
Lets loop through every page here to ensure all usages of the image component are valid HTML

Reply:
I will try but that requires a lot of changes to these pages because a lot of them are not valid W3C because of all kinds of errors like duplicated IDs etc. And I feel like it's unrelated to this PR.

Testing only with the `/valid-html-w3c` page keeps things simple and only test what is really needed.
Will keep you updated! :+1: